app/src/store/actionCreators.js

```diff
@@ -2,29 +2,53 @@ import { actionTypes } from './actionTypes';
 
 const globalUrl = 'http://localhost:4000';
 
-function getMovies(dispatch, getState){
-    var url = globalUrl + '/movies';
+function getMovies(search) {
+    return (dispatch, getState) => {
+        var url = globalUrl + '/movies';
+        var useQuestion = true;
 
-    var state = getState();
+        var state = getState();
 
-    var filter = state.filter.selectedOption.name.toLowerCase();
-    if(filter && filter != 'all'){
-        url += `?filter=${filter}`;
-    }
+        var filter = state.filter.selectedOption.name.toLowerCase();
+        if(filter && filter != 'all'){
+            url += `?filter=${filter}`;
+            useQuestion = false;
+        }
+
+        var sorter = state.sorter.selectedOption.field.toLowerCase();
+        if(sorter){
+            var sortOrder = state.sorter.asc ? 'asc' : 'desc';
+            url += (useQuestion ? '?' : '&') + `sortBy=${sorter}&sortOrder=${sortOrder}`;

```

Comment:
create and use everywhere helper that will return such string: 

`?a=a&b=b`
it's quite common

Reply:
Done)